samples/nrf9160/asset_tracker/src/main.c

```diff
@@ -64,20 +70,20 @@
 	* defined(CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES)
 	*/
 
-enum {
-#if defined(CONFIG_BOARD_NRF9160_PCA20035)
+static enum {
```

Comment:
I guess this stuff will disappear when it's rebased on top of master

Reply:
Not sure I understand, what do you mean?

Reply:
Oops, this was just me comparing to some outdated code